### ISSUE TITLE
Feature request: Possibility for interaction

### ISSUE THREAD

**Issue by bardt** [kulkalkul/bevy_mod_billboard#14]
I need a possibility to track hover and press events on a billboard. 

I tried to achieve it using `bevy_mod_picking` crate and exposing the meshes like this:
```rust
fn add_billboard_mesh(
    mut commands: Commands,
    billboards: Query<(Entity, &BillboardMeshHandle), Added<BillboardMeshHandle>>,
) {
    for (entity, mesh_handle) in billboards.iter() {
        let handle = mesh_handle.0.clone();
        commands.entity(entity).insert(handle);
    }
}
```

but meshes from BillboardMeshHandle are positioned on screen differently from where the billboard would be rendered, and raycasting triggers in a wrong screen area:

<img width="1392" alt="image" src="https://github.com/kulkalkul/bevy_mod_billboard/assets/633908/806602c8-ce68-4ed0-8aa8-c2cbb41641ec">

**Comment by kulkalkul** [comment]
Yeah that's normal, because only the position and the scale of the object is used in the shader, rotation is calculated in shader itself. So bevy mod picking doesn't have that rotation information.

As for feature request; I don't know how complicated it is to implement such thing. I need to investigate how bevy mod picking implements it. If it is complicated, I might pass on it. Right now my to-do list is big, so I can't guarantee it anytime soon.

In the meantime, for now, an easier approach would be using CPU managed text objects with Transform::look_at calls with more basic text widget, like this implementation: https://github.com/bevyengine/bevy/pull/7426/files
